PR-Trigger/index.ts

```diff
@@ -37,34 +34,31 @@ const httpTrigger = async function (context, _req) {
 
     // Allow the bot to run side-effects which are not the 'core' function
     // of the review cycle, but are related to keeping DT running smoothly
-    const sideEffects = {
-        "check_suite": mergeCodeOwnersOnGreen
+    if (event === "check_suite") {
+        await mergeCodeOwnersOnGreen(req.body as EventPayloads.WebhookPayloadCheckSuite);

```

Comment:
Good call on some of these simplifications - now we know what this codebase is, we don't really need it

Reply:
👍